core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/constants/UserStoreUIConstants.java

```diff
@@ -0,0 +1,103 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.wso2.carbon.user.core.constants;
+
+import org.wso2.carbon.user.api.Property;
+
+/**
+ * Meta constants which are required to populate the configuration UIs.
+ */
+public class UserStoreUIConstants {
+
+    public static final String TYPE = "type";
+
+    public static final String CATEGORY = "category";
+
+    public static final String REQUIRED = "required";
```

Comment:
shall we make this 'isRequired' ?

Reply:
Also shall we add another property for 'isConfidential'  for properties we need to mask the values?

Reply:
> shall we make this 'isRequired' ?

Since this for the UI manipulation.. 'required' seems more suitable..